ui/packages/catalog-ui-search/src/main/webapp/component/location-old/location-old.js

```diff
@@ -76,7 +76,8 @@ function convertToValid(key, model) {
     key.lon = Math.min(180, key.lon)
   }
   if (key.radius !== undefined) {
-    key.radius = Math.max(minimumBuffer, key.radius)
+    let tempRadius = Math.max(minimumBuffer, key.radius)
```

Comment:
✏️ `const`
Also, could we name this something that does more to distinguish it from `model.get('radius')`? I don't know what the difference between the two is just by looking at the code.